fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.yaml

```diff
@@ -0,0 +1,40 @@
+id: fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
```

Comment:
The app has extension point `fr.free.hxc2001.HxCFloppyEmulator.Extension` with `subdirectories: false`. This means there can be only one possible extension with ID exactly `fr.free.hxc2001.HxCFloppyEmulator.Extension`.
For this to work, either remove `.capslib` portion here or set `subdirectories` to `true` in the app manifest.